src/frontend/packages/lib_components/src/common/Card/ClosingCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Box,
+  Text,
+  Card as GrommetCard,
+  CardBody,
+  BoxTypes,
+  Button,
+} from 'grommet';
+import { PropsWithChildren, useState } from 'react';
+import styled from 'styled-components';
+
+const CardLayout = styled(GrommetCard)`
+  transition: all 0.3s;
+`;
+
+interface ClosingCardProps extends BoxTypes {
+  title: string;
+}
+
+export const ClosingCard = ({
+  title,
+  children,
+  ...cardLayoutProps
+}: PropsWithChildren<ClosingCardProps>) => {
+  const [display, setDisplay] = useState(true);
+
+  const hide = {
+    padding: 0,
+    margin: 0,
+    height: 0,
+    opacity: 0,
+  };
+
+  return (
+    <CardLayout
+      height="inherit"
+      width="xsmedium"
+      background="white"
+      round="xsmall"
+      elevation="small"
+      {...cardLayoutProps}
+      style={
+        display ? cardLayoutProps.style : { ...cardLayoutProps.style, ...hide }
+      }
+      role="alert"
+      aria-hidden={!display}
+    >
+      <CardBody pad={{ horizontal: 'medium', vertical: 'xsmall' }}>
+        <Box direction="row" justify="between" align="center">
+          <Text size="small" weight="bold">
+            {title}
+          </Text>
+          <Button onClick={() => setDisplay(false)}>✖</Button>
```

Comment:
Maybe I'm wrong, but to we use this kind of char elsewhere ?

Reply:
[fixup! ✨(lib-components) create ClosingCard component](https://github.com/openfun/marsha/pull/2240/commits/c20aa1a559f11363e92f0723aec0f95e2296b90d)